doc/api_assets/hljs.css

```diff
@@ -7,6 +7,7 @@
   color: #333;
 }
 
+.hljs-attribute,
```

Comment:
Is this going to have any unintended affect on the other existing code blocks?

Reply:
Nope, these are the only two instances I was able to find. Something that does not yet have highlighting is the `.hljs-attr` class, which probably needs to happen at some point, but this is not the same as that.